pkg/agent/types/storage.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+// Stores are named identifiers corresponding to key-value buckets
+type Store string
+
+const (
```

Comment:
Not sure if I like these defined in `types`, since they're not actually types. Open to suggestions of better arrangements.